lib/screens/cart_screen.dart

```diff
@@ -0,0 +1,199 @@
+import 'package:flutter/material.dart';
+import 'package:furniture/commons/reusable_button.dart';
+import 'package:furniture/commons/reusable_counter.dart';
+
+class CartScreen extends StatefulWidget {
+  const CartScreen({
+    super.key,
+    required this.image,
+    required this.name,
+    required this.totalPrice,
+    required this.quantity,
+    required this.actualPrice,
+  });
+
+  final String? image;
+  final String? name;
+  final double? totalPrice;
+  final int quantity;
+  final double actualPrice;
```

Comment:
These all are properties of a product so it's better to directly take product object as a param instead of taking individual properties